src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/UpdateGithubIssueTest.cs

```diff
@@ -119,5 +120,174 @@ public Issue GetIssue()
             );
             return issue;
         }
+
+        [Fact]
+        public async Task CreateIssueAndAddAdditionalComment()
+        {
+            RepositoryBranchUpdateHistoryEntry firstError =
+                new RepositoryBranchUpdateHistoryEntry
+                {
+                    Repository = RepoUrl,
+                    Branch = Branch,
+                    Method = MethodName,
+                    Timestamp = new DateTime(2200, 1, 1),
+                    Arguments = $"[\"{SubscriptionId}\",\"{MethodName}\",\"{ErrorMessage}\"]",
+                    Success = false,
+                    ErrorMessage = ErrorMessage,
+                    Action = "Creating new issue"
+                };
+            RepositoryBranchUpdateHistoryEntry secondError =
+                new RepositoryBranchUpdateHistoryEntry
+                {
+                    Repository = RepoUrl,
+                    Branch = Branch,
+                    Method = "ProcessPendingUpdatesAsync",
+                    Timestamp = new DateTime(2200, 2, 1),
+                    Arguments = "ProcessPendingUpdatesAsync error",
+                    Success = false,
+                    ErrorMessage = ErrorMessage,
+                    Action = "Create a new issue comment",
+                };
+
+            Context.RepoBranchUpdateInMemory = new List<RepositoryBranchUpdateHistoryEntry>
+                {firstError , secondError};
+            Maestro.Data.Models.Subscription subscription = new Maestro.Data.Models.Subscription
+            {
+                Id = Guid.Parse(SubscriptionId),
+                SourceRepository = "Source Repo",
+                TargetRepository = "Target Repo",
+            };
+            Context.Subscriptions.Add(subscription);
+            Context.SaveChanges();
+            Repository repository = new Repository();
+            GithubClient.Setup(x => x.Repository.Get(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(repository);
+            Issue updateIssue = GetIssue();
+            Octokit.AuthorAssociation author = new Octokit.AuthorAssociation();
+            IssueComment comment = new IssueComment
+            (
+                1,
+                null,
```

Comment:
Add parameter names to the arguments here, I don't know what the 6 nulls are supposed to be.